buildconfig/pygame-stubs/mask.pyi

```diff
@@ -59,24 +55,24 @@ class Mask:
     def count(self) -> int: ...
     def centroid(self) -> Tuple[int, int]: ...
     def angle(self) -> float: ...
-    def outline(self, every: Optional[int] = 1) -> List[Tuple[int, int]]: ...
+    def outline(self, every: int = 1) -> List[Tuple[int, int]]: ...
     def convolve(
         self,
         other: Mask,
         output: Optional[Mask] = None,
-        offset: Optional[_Coordinate] = (0, 0),
+        offset: _Coordinate = (0, 0),
     ) -> Mask: ...
     def connected_component(
         self, pos: Union[List[int], Tuple[int, int]] = (-1, -1)
```

Comment:
You didn't change this or anything, but this looks fishy to me.

It's not that the pos is a default argument, it really is just optional. https://www.pygame.org/docs/ref/mask.html#pygame.mask.Mask.connected_component.

Would passing in (-1,-1) really be the same as passing nothing? The docs don't say so.

Maybe this is a good spot for a good old `@overload` instead.

Reply:
Yes, going through the code, I see that passing `(-1, -1)` would actually error, but if no value is passed, the internal functions get `(-1, -1)`

In any case, yeah you are right, we can make the typehint something like `pos: Union[List[int], Tuple[int, int]] = ...` and not `(-1, -1)`